app/models/katello/ping.rb

```diff
@@ -5,20 +5,25 @@ class Ping
     PACKAGES = %w(katello candlepin pulp qpid foreman tfm hammer).freeze
 
     class << self
+      def pulpcore_enabled # for downstream 6.9, remove in 6.10
+        SETTINGS[:katello][:use_pulp_2_for_content_type].nil? || (!SETTINGS[:katello][:use_pulp_2_for_content_type][:yum] &&
+        !SETTINGS[:katello][:use_pulp_2_for_content_type][:docker] &&
+        !SETTINGS[:katello][:use_pulp_2_for_content_type][:file]) ||
+        system('systemctl is-enabled pulpcore-api.service')
```

Comment:
I know this is already merged, but I'd think that should fail with SELinux enabled, as I don't think Apache/Puma can run `systemctl`?

Reply:
When I was in pry, I was able to see the output of `system('systemctl is-enabled pulpcore-api.service')` and `true` `false` come back correctly.

Reply:
Was that in a prod setup with selinux in enforcing mode? Anyways, just a hunch. We'll see if that fails in prod early enough ;)

Reply:
hehe. The pry was in my devel setup. I did test it also with prod vagrant nightly without a debugger and got the ping output to change.